engine/plugins/state_machine/action_handler.js

```diff
@@ -247,6 +249,36 @@ class ActionHandler {
         });
     }
 
+    /**
+     * Get item from client.actionData.
+     *
+     * Applies 2of5 transformation of itemIdentifier.
+     *
+     * @param client
+     *
```

Comment:
Missing parameter and return descripions?

Reply:
Fixed